pkg/operator/ceph/cluster/rbd/controller_test.go

```diff
@@ -192,6 +193,7 @@ func TestCephRBDMirrorController(t *testing.T) {
 			context:          c,
 			peers:            make(map[string]*peerSpec),
 			opManagerContext: ctx,
+			recorder:         record.NewFakeRecorder(5),
 		}
 
 		rbdMirror.Spec.Peers.SecretNames = []string{peerSecretName}
```

Comment:
We should verify here that the fake recorder registered our event with `assert.Contains(t, <-fakeRecorder.Events, "part of our expected message")`

The `fakeRecorder` will have to be kept in its own variable as well.